NewForce-Cohort-6/tabloidfullstack-stuffing
# SB/Fix-Authorization-Bug

This is a band-aid fix for the authorization bug. It double-checks if there's a user in local storage and redirects accordingly.

To test

1. While logged out ensure that the home directory navigates you to the "/login " and loads the login component
2. While logged in, ensure that you can see the subscription list from the home directory
3. While logged in, ensure that a non-existing uri route leads you back to the home directory "/"
